1.Core/UIElements/Interfaces/ITypeable.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Todoly.Core.UIElements.Interfaces
 {
-    public interface ITypeable
+    public interface ITypeable : IElement
```

Comment:
Same as IClickable, it could be an ITextInput interface

Reply:
Done